src/Weaviate.Client/WeaviateClient.cs

```diff
@@ -157,6 +157,43 @@ public class WeaviateClient : IDisposable
 
     public System.Version WeaviateVersion => GetWeaviateVersionAsync().GetAwaiter().GetResult();
 
+    /// <summary>
+    /// Returns true if the Weaviate process is live.
+    /// </summary>
+    public Task<bool> Live() => RestClient.LiveAsync();
+
+    /// <summary>
+    /// Returns true if the Weaviate instance is ready to accept requests.
+    /// </summary>
+    public Task<bool> IsReady() => RestClient.ReadyAsync();

```

Comment:
could you add a small test with a running weaviate? Its ok if you only test the "is ready/live" case